gmso/external/convert_hoomd.py

```diff
@@ -95,15 +95,47 @@ def to_gsd_snapshot(
     Return
     ------
     gsd_snapshot : gsd.hoomd.Frame
-        Converted hoomd Snapshot.
+        Converted hoomd Snapshot. Always retruned.
     base_units : dict
-        Based units dictionary utilized during the conversion.
+        Based units dictionary utilized during the conversion. Always returned.
+    rigid_info : hoomd.md.constrain.Rigid
+        Hoomd object storing constituent particle information
+        needed to run rigid body simulations in Hoomd-Blue.
+        This is only returned if Molecule's with `Molecule.isrigid = True`
+        are found in the topology.
 
     Notes
     -----
-    Force field parameters are not written to the GSD file and must be included
-    manually in a HOOMD input script. Work on a HOOMD plugin is underway to
-    read force field parameters from a Foyer XML file.
+    Force field parameters are not written to the GSD file and must be
+    generated using `to_hoomd_forcefield()`.
+
+    If you are using mBuild and GMSO to initialize rigid body simulations
+    with Hoomd-Blue set the site's `Molecule.isrigid` property to `True`.
+    If your topology contains a mix of rigid and flexible molecules,
+    the rigid molecules must come first in the heirarchy of the GMSO toplogy
+    and therefore also in the mBuild compound heirarchy.
+    See https://hoomd-blue.readthedocs.io/en/latest/index.html
+    for more information about running rigid body simulations in Hoomd-Blue.
+
```

Comment:
```suggestion
    For more information on the units from `base_unites`, see https://hoomd-blue.readthedocs.io/en/latest/units.html#base-units
    
```

Reply:
I'm not sure what you mean here. This is part of a Note about how to correctly set up rigid simulations, and shouldn't need to refer to base units at all. I updated the link to go directly to one of the rigid body tutorials in the hoomd docs though.

Reply:
Well the base_units is one of the main return objects that you'll need to run the simulations no? So just some extra reference in case you want to interpret this object. It's a separate note from the rigid bodies note.

Reply:
Good idea. I didn't notice the change was just an addition.